src/shared/GlobalTopBanners/GlobalTopBanners.tsx

```diff
@@ -1,10 +1,22 @@
 import { lazy } from 'react'
+import { useRouteMatch } from 'react-router-dom'
 
 import SilentNetworkErrorWrapper from 'layouts/shared/SilentNetworkErrorWrapper'
 
 const SentryTrialBanner = lazy(() => import('./SentryTrialBanner'))
+const RequestInstallBanner = lazy(() => import('./RequestInstallBanner'))
 
 const GlobalTopBanners: React.FC = () => {
+  const ownerMatch = useRouteMatch('/:provider/:owner')
+
+  if (ownerMatch?.isExact) {
+    return (
+      <SilentNetworkErrorWrapper>
+        <RequestInstallBanner />
+      </SilentNetworkErrorWrapper>
+    )
+  }
+
```

Comment:
We may want to show multiple banners at the same time, can you move the owner match check into the banner itself? That way we don't bloat up `GlobalTopBanners` with a bunch of conditional checks. In the `RequestInstallBanner` just check return `null` if it is an exact match.

Reply:
As well if we want to check routes for any other banners, this limits us to only this single route check.

Reply:
tbh it's still weird to me that we place a non-global banner that belongs to a specific page within global banners component just to match the route, but at least we don't mess with old banner here

Reply:
So you can put banners in here that will be displayed globally however, that is not a requirement. But it allows us to co-locate all banners that will be displayed at the top of the page outside of the `<main>` tag where they should not be located as they're not the main content of the page.